go/libraries/doltcore/sqle/sqlutil/convert.go

```diff
@@ -32,14 +33,26 @@ func FromDoltSchema(tableName string, sch schema.Schema) (sql.PrimaryKeySchema,
 	var i int
 	_ = sch.GetAllCols().Iter(func(tag uint64, col schema.Column) (stop bool, err error) {
 		sqlType := col.TypeInfo.ToSqlType()
-		var extra string
-		if col.AutoIncrement {
-			extra = "auto_increment"
-		}
 
 		var deflt *sql.ColumnDefaultValue
 		if col.Default != "" {
-			deflt = sql.NewUnresolvedColumnDefaultValue(col.Default)
+			if col.Default == "NULL" {
```

Comment:
I'm a little confused by this. Shouldn't this be in the engine in Eval() of an UnresolvedComment. Or another way to look at it is that are we doing the conversion of a SQL Schema to a Dolt Schema incorrectly?